### PR TITLE
deprecate RRPacking in default config

### PR DESCRIPTION
**Pull request by huijunw**
use ResourceCompliantRRPacking as default in config to make re-packing compatible with packing.

ps: safest is to use the same alg for both packing and repacking

### CODE REVIEW

File: heron/config/src/yaml/conf/aurora/packing.yaml [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # packing algorithm for packing instances into containers
-heron.class.packing.algorithm:    com.twitter.heron.packing.roundrobin.RoundRobinPacking
+heron.class.packing.algorithm:    com.twitter.heron.packing.roundrobin.ResourceCompliantRRPacking
+heron.class.repacking.algorithm:  com.twitter.heron.packing.binpacking.FirstFitDecreasingPacking
```
Comment (kramasamy) [inline heron/config/src/yaml/conf/aurora/packing.yaml]: @huijunw - thanks for the PR - should we use the same algorithm for both?

Comment (huijunw) [reply heron/config/src/yaml/conf/aurora/packing.yaml]: done

**Comment by nwangtw** [comment]
LGTM.

Related question: if RoundRobin is not good, should we deprecate it? Like removing it or adding a warning when user uses it? I am wondering if there are some cases that RoundRobin is better than the others.

**Comment by huijunw** [comment]
I would prefer keeping the RR alg since it is tested in production and simple enough as a reference for new developers.

**Comment by nwangtw** [comment]
SGTM. Thanks

**Pull request merged**